_includes/groups/_meeting-details.html

```diff
@@ -14,5 +14,5 @@
   {% endif %}
 </p>
 {% if meeting.registration_link %}
-  <p class="push-top"><a href="{{ meeting.registration_link }}" class="btn btn-cyan" type="button">Register</a></p>
+  <p class="push-top"><a href="{{ meeting.registration_link }}" class="btn btn-orange-blue" type="button">Register</a></p>
```

Comment:
Where is `btn-orange-blue` defined?

Reply:
Resolved on crds-styles. 
[CRDS-STYLES](https://github.com/crdschurch/crds-styles/pull/429)

Reply:
https://github.com/crdschurch/crds-styles/pull/432